src/backends/test-repo/implementation.js

```diff
@@ -26,7 +26,7 @@ function nameFromEmail(email) {
 export default class TestRepo {
   constructor(config) {
     this.config = config;
-    this.assets = [];
+    this.assets = window.repoFiles.assets || [];
```

Comment:
Is this line related to this PR at all?

Reply:
https://github.com/netlify/netlify-cms/pull/786/files#diff-1634bb1b7f4da7ca72cddcb74fef74b7R12

Allows for the loading of the assets that are truly in the `assets/images` folder. The way they would show up if you were truly hitting a real api.

Reply:
Awesome!